quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -420,6 +420,16 @@ public void onParticipantConnected(Room room, Participant participant) {
             public void onParticipantDisconnected(Room room, Participant participant) {
                 removeParticipant(participant);
             }
+
+            @Override
+            public void onRecordingStarted(Room room) {
+
```

Comment:
Actually the comment on the changelog is more accurate. So perhaps a paraphrase of that.

> Added an isRecording method to Room, and callbacks to RoomListener. Please note that recording is only available in our Group Rooms developer preview. isRecording will always return false in a P2P Room.